### ISSUE TITLE
Extract xyz data from tensile and pressure fields similar to nodallines?

### ISSUE THREAD

**Issue by BenjiDa** [obspy/obspy#1640]
Obspy 1.0.2
Python 2.7.12
OSX 10.10.5
Installed ObsPy via Anaconda

Not an issue so much as a question. I am trying to extract data for plotting in a 3d model, 

```
mt = [7.982,-0.154, -7.574, 0.548, 7.147, -0.211] 

mopad_mt = MomentTensor(mt, system='NED')
bb = BeachBall(mopad_mt, npoints=200)
bb._setup_BB(unit_circle=False)

 # extract the coordinates of the nodal lines
neg_nodalline = bb._nodalline_negative


```

Here are results as I plot them up using mayavi: 
<img width="581" alt="screen shot 2017-01-10 at 2 36 42 pm" src="https://cloud.githubusercontent.com/assets/18178879/21827954/b69e8f64-d742-11e6-8e9c-4944d09f1890.png">

Is there away to extract data for tensile and pressure fields (e.g. xyz positions) similar to nodal lines? 

Thanks

**Comment by krischer** [comment]
Please also have a look at this PR: https://github.com/obspy/obspy/pull/1192

**Comment by MMesch** [comment]
you should check out this code: https://github.com/obspy/obspy/blob/master/obspy/imaging/source.py#L73-L598

( careful, there is a small bug that is fixed in #1553 )

**Comment by megies** [comment]
stale, closing.
